src/com/dotmarketing/util/Config.java

```diff
@@ -510,6 +510,8 @@ public static void _setupFakeTestingContext() throws Exception{
 		// if we need a fake ServletContext
 		if(CONTEXT ==null){
 			ServletContext context = Mockito.mock(ServletContext.class);
+			Mockito.when(context.getInitParameter("company_id")).thenReturn("dotcms.org");
```

Comment:
is it not a way to encapsulate this in a separated class. May be if we give protected access to the dotmarketingPropertiesUrl and clusterPropertiesUrl properties, a diff class in the same package on the test sources could init the dotmarketingPropertiesUrl  and clusterPropertiesUrl, what do you think?
It could reduce the accomplish :)